src/TLuaInterpreter.cpp

```diff
@@ -12276,8 +12286,11 @@ void TLuaInterpreter::initLuaGlobals()
     lua_register(pGlobalLua, "getColumnCount", TLuaInterpreter::getColumnCount);
     lua_register(pGlobalLua, "getRowCount", TLuaInterpreter::getRowCount);
     lua_register(pGlobalLua, "getOS", TLuaInterpreter::getOS);
+    lua_register(pGlobalLua, "setRoomSymbol", TLuaInterpreter::setRoomSymbol);
+    lua_register(pGlobalLua, "getRoomSymbol", TLuaInterpreter::getRoomSymbol);
```

Comment:
Don't need this bit anymore.

Reply:
Yes you do because that is what allows the better names using the word "symbol" to be used as well as (what I want to deprecate) "char" which isn't even really a word IMVHO - both work and call the *same* functions - in the same way as `clearWindow` and `clearUserWindow` do.

Reply:
I don't think `clearWindow` and `clearUserWindow` is a mistake we want to repeat again.

Let's look at it from the perspective of the users, and not a developer-centric purity point of view. It's a lot simpler for the users if there's one function that does one thing instead of two that do one thing?

Imagine this conversation: "What's this function B?" "Oh it does the same thing as function A." "Oh right... that makes total sense."

Reply:
> I don't think clearWindow and clearUserWindow is a mistake we want to repeat again.

I am not sure what you mean by that - perhaps it predates my starting on the Mudlet project, but this does provide a means to get past the fact that the original functions were perhaps poorly named or at least not really appropriate when their scope had been expanded beyond their original target.

As the Wiki seems to be the goto documentation for the lua API it is well within the scope of things to have the original functions be stubs that say, ***see \*etRoomSymbol(...)*** and to explain in the latter that since **Mudlet 3.8.0** the functions' capability are extended.

You have always said that user/scripters should test for the presence of a function to determine it's availability and like this they can do so - without us having to have both.  To work with just a single ASCII character in a backwards compatible manner they can use the old names - and existing stuff will continue to work (which is *essential* - I've learnt **that** by now!) but going forward new users/scripters in the future will be guided to use the more appropriately named "symbol" ones - and if anyone asks we can say that is why there are synonymous functions ...

Reply:
I thought we already agreed there wouldn't be any backwards compatibility breaking already by upgrading roomchar's capability. What are you going for - to have roomchar be artificially limited to 1 ASCII character while roomsymbol is any unicode text?

Reply:
There is now no separate `*etRoomChar(`...`)` functions - only the new functions are in `TLuaInterpreter.cpp` but both sets are registered so that the old names still work in Lua scripts but called the new ones `*etRoomSymbol(`...`)` internally.

📖  I will edit the Wiki as required when this PR goes in - explaining that the revised functionality can be tested for against the new names (if that is *really* needed by scripters) but that the old names are left to preserve backwards compatibility...